python/dolfinx/wrappers/geometry.cpp

```diff
@@ -180,13 +180,18 @@ void declare_bbtree(nb::module_& m, std::string type)
       nb::arg("mesh"), nb::arg("dim"), nb::arg("indices"), nb::arg("points"));
   m.def("determine_point_ownership",
         [](const dolfinx::mesh::Mesh<T>& mesh,
-           nb::ndarray<const T, nb::c_contig> points, const T padding)
+           nb::ndarray<const T, nb::c_contig> points,
+           nb::ndarray<const std::int32_t, nb::ndim<1>, nb::c_contig> cells,
+           const T padding)
         {
           const std::size_t p_s0 = points.ndim() == 1 ? 1 : points.shape(0);
           std::span<const T> _p(points.data(), 3 * p_s0);
           return dolfinx::geometry::determine_point_ownership<T>(mesh, _p,
+                                                                 std::span(cells.data(), cells.size()),
                                                                  padding);
-        });
+        },
+      nb::arg("mesh"), nb::arg("points"), nb::arg("cells"), nb::arg("padding") = 0.0,
```

Comment:
I guess what @jhale meant is that the C++ function in `utils.h` doesn't have a default, and that should be made consistent.

Reply:
We avoid default args in the nanobind layer.